### ISSUE TITLE
renderProp does not work

### ISSUE THREAD

**Issue by olabalboa** [wojtekmaj/enzyme-adapter-react-17#24]
I have tried using this enzyme adapter in our codebase with React 17 and all tests passes fine except the ones using `renderProp`.
The following test (simple version of this example https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/renderProp.html) works fine with React 16 and enzyme-adapter-react-16 but not when using ver 17

```javascript
  const App = () => <Component render={() => <div>render</div>} />;
  const Component = () => <div />;

  it('should render prop', () => {
    const wrapper = shallow(<App />)
      .find(Component)
      .renderProp('render')();
    expect(wrapper.equals(<div>render</div>)).toEqual(true);
  });
```

**Comment by henryqdineen** [comment]
@wojtekmaj We ran into this and I believe https://github.com/wojtekmaj/enzyme-adapter-utils/pull/1 should fix it.

**Comment by wojtekmaj** [comment]
Fixed in 0.6.2.
